### PR TITLE
OH2-246 Add OpenAPI specs with action for update upon PR (with Maven)

### PR DESCRIPTION
**Pull request by mwithi**
See OH2-246.

Option 2+3.

### CODE REVIEW

**Review by ArnaudFonzam — APPROVED** [review]

File: src/main/java/org/isf/accounting/rest/BillController.java [hunk 1]
```diff
@@ -369,9 +370,10 @@ public ResponseEntity<List<BillDTO>> getPendingBills(@RequestParam(value="patien
 	 * @throws OHServiceException
 	 */
 	@PostMapping(value = "/bills/search/by/item", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<BillDTO>> searchBills(@RequestParam(value="datefrom") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateFrom,
-			@RequestParam(value="dateto")@DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateTo,
-			@RequestBody BillItemsDTO billItemDTO) throws OHServiceException {
+	public ResponseEntity<List<BillDTO>> searchBills(
```
Comment (ArnaudFonzam) [inline src/main/java/org/isf/accounting/rest/BillController.java:373]: please why we have space here ?

File: src/main/java/org/isf/accounting/rest/BillController.java [hunk 2]
```diff
@@ -202,9 +202,10 @@ public ResponseEntity<FullBillDTO> updateBill(@PathVariable Integer id, @Request
 	 * @throws OHServiceException
 	 */
 	@GetMapping(value = "/bills", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<BillDTO>> searchBills(@RequestParam(value = "datefrom") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateFrom,
-			@RequestParam(value = "dateto") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateTo,
-			@RequestParam(value = "patient_code", required = false, defaultValue = "") Integer code) throws OHServiceException {
+	public ResponseEntity<List<BillDTO>> searchBills(
```
Comment (ArnaudFonzam) [inline src/main/java/org/isf/accounting/rest/BillController.java:205]: also here we have a big space

Comment (mwithi) [reply src/main/java/org/isf/accounting/rest/BillController.java:205]: it's the formatter doing that (`...line_length" value="160"/>`)

File: src/main/java/org/isf/accounting/rest/BillController.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/accounting/rest/BillController.java:373]: it's the formatter doing that (`...line_length" value="160"/>`)

**Pull request merged**